content/user-guide/projects.md

```diff
@@ -21,6 +23,24 @@ There are four roles with different levels of access rights that can be assigned
 - **Commenter:** the user cannot edit folders or diagrams nor invite users, but can leave [comments](../diagrams#comments) on each diagram.
 - **Viewer:** the user cannot edit folders or diagrams nor leave comments, but only view diagrams.
 
+The **Project Admin** is the only user entitled to invite and manage user roles.
+
+On the Home Page you are able to easily identify who has access to the project:
+
+{{<img src="../default-project.png" title="Project created from Cawemo - collaborators visibility" width="150%" height="150%">}}
+
+{{< note class="info" >}}
+When there are more than a handful of users, you can hover over the circle with a number on it to view the names of all users that have access.
+{{< /note >}}
+
+### Projects used by integrations
+
+Projects used by integrations do not allow you to manage access rights.
+All [members of your organization](../settings#managing-members-admin-only) automatically gain _read-only_ access to your catalog, engine and modeler projects, meaning that they can see your files, but not edit them.
+
+On the Home Page you are able to easily identify who has access to the project. In this case: **Everyone** in the organization has access:
```

Comment:
```suggestion
On the Home Page you are able to easily identify who has access to the project. 
In this case **everyone** in the organization has access:
```

Reply:
Changes done in https://github.com/camunda/camunda-docs-cawemo/pull/184/commits/5eb039b821aa7fa664be1bdb5c8e1f8d88433c2b